zerver/migrations/0319_create_edit_message_array_and_backfill_edit_messages.py

```diff
@@ -0,0 +1,49 @@
+# Generated by Django 3.1.7 on 2021-04-14 11:26
+import django.contrib.postgres.fields
+import orjson
+from django.db import migrations, models
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def backfill_edit_messages(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Message = apps.get_model("zerver", "Message")
+    ArchivedMessage = apps.get_model("zerver", "ArchivedMessage")
+    msg_models = [Message, ArchivedMessage]
+
+    for model in msg_models:
+        for message in model.objects.all():
+            edit_history = message.edit_history
+
+            if edit_history is not None:
+                edit_history_array = []
+                for edit in edit_history:
+                    edit_history_array.append(orjson.loads(edit))
+
+                message.edit_history_array = edit_history_array
+                message.save(update_fields=["edit_history_array"])
```

Comment:
How have you tested this migration?

Reply:
I haven't written specific test for this commit, I just ensured that the existing tests pass and the edit message functionality is working as expected

Reply:
Ah sorry! I misunderstood the question, I have tested it in the shell... I printed the content of the `edit_history` field and the contents of the `edit__history_array` field, and verified manually that the contents are same and the content of the `edit_history_array` are deserialzed JSON stored in a list.